<?xml version="1.0"?>
<launch>

<include file="$(find om_modbus_master)/launch/om_modbusRTU.launch"/>

<node pkg="om_cart" name="om_cart" type="om_cart">
    <remap from="cmd_vel" to="cmd_raw"/>
</node>


<!--map load-->
<arg name="map" default="outdoor_0212_1.yaml"/>
<!--map server-->
<node pkg="map_server" type="map_server" name="map_server" args="$(find hdl_localization)/data/$(arg map)">
</node>

<include file="$(find hdl_localization)/launch/hdl_localization_livox.launch" />

<param name="waypoint_path" value="$(find amr_ros)/path/waypoints.txt"/>
<node name="load_waypoints" pkg="amr_ros" type="load_waypoints.py" output="screen"/>

<!--trajectry -->
<include file="$(find pure_pursuit)/launch/pure_pursuit_node.launch" />

<!--Start 3DLidar -->
<include file="$(find livox_ros_driver2)/launch_ROS1/rviz_MID360.launch" />

<arg name="gui" default="true" />
<param name="use_gui" value="$(arg gui)"/>
<arg name="rvizconfig" value="$(find amr_ros)/rviz/path_fixed.rviz"/>
<node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />


</launch>
